storage/gcp/gcp.go

```diff
@@ -178,17 +178,29 @@ func (s *Storage) Add(ctx context.Context, e *tessera.Entry) tessera.IndexFuture
 	return s.queue.Add(ctx, e)
 }
 
-// Get returns the requested object.
+func (s *Storage) ReadCheckpoint(ctx context.Context) ([]byte, error) {
+	return s.get(ctx, layout.CheckpointPath)
+}
+
+func (s *Storage) ReadTile(ctx context.Context, l, i, sz uint64) ([]byte, error) {
+	return s.get(ctx, layout.TilePath(l, i, sz))
+}
+
+func (s *Storage) ReadEntryBundle(ctx context.Context, i, sz uint64) ([]byte, error) {
+	return s.get(ctx, layout.EntriesPath(i, sz))
+}
+
+// get returns the requested object.
```

Comment:
```suggestion
// Get returns the requested object.
```

Reply:
I did that to match the newly-unexported method name (same style as on `init` below).

Reply:
I must have read the diff backwards or something...